src/shims/ApiCompatBaseline.uapaot.netstandard.txt

```diff
@@ -3,6 +3,14 @@ TypeCannotChangeClassification : Type 'System.RuntimeArgumentHandle' is a 'ref s
 TypeCannotChangeClassification : Type 'System.TypedReference' is a 'ref struct' in the implementation but is a 'struct' in the contract.
 TypeCannotChangeClassification : Type 'System.IO.FileAttributes' is a 'class' in the implementation but is a 'struct' in the contract.
 Compat issues with assembly netstandard:
+MembersMustExist : Member 'System.Runtime.CompilerServices.RuntimeHelpers.GetSubArray(T[], System.Range)' does not exist in the implementation but it does exist in the contract.
```

Comment:
Looks like you're missing the '<T>' from this method signature.
```
MembersMustExist : Member 'System.Runtime.CompilerServices.RuntimeHelpers.GetSubArray<T>(T[], System.Range)' does not exist in the implementation but it does exist in the contract.
```

Reply:
Weird, the original source I copied this from didn't have the `<T>`: https://github.com/dotnet/corefx/pull/35572#issuecomment-467565178. I'll fix & update